torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -361,11 +361,12 @@ void export_opnames(const script::Module& m, std::set<std::string>& opnames) {
   moduleMethodsTuple(m, elements);
   for (const auto& element : elements) {
     auto table = element.toTuple()->elements()[1];
-    auto row = table.toTuple()->elements().at(BYTECODE_INDEX_OPERATOR).toTuple();
+    auto row =
+        table.toTuple()->elements().at(BYTECODE_INDEX_OPERATOR).toTuple();
     TORCH_INTERNAL_ASSERT(
-      row->elements().at(0).toStringRef() == "operators",
-      "Expected operators but found ",
-      row->elements().at(0).toStringRef());
+        row->elements().at(0).toStringRef() == "operators",
+        "Expected operators but found ",
+        row->elements().at(0).toStringRef());
```

Comment:
Should this change be here?

Reply:
this is clang format changes, got this when I run clang-format-all.py